cdap-docs/developer-manual/source/building-blocks/schedules.rst

```diff
@@ -8,169 +8,247 @@
-  schedule was triggered |---| the Stream has ingested more than the increment of data defined by the schedule,
-  the workflow will be immediately executed upon resume.
-- *Update*: this action is triggered when an application which contains a schedule is redeployed in CDAP.
-  If the schedule name has not been modified and one of its parameters has changed |---| such as the `dataTrigger` |---| the
-  *update* action will be called. If the `dataTrigger` is updated, and the Stream has ingested more data than defined
-  by the `dataTrigger` since the last time the schedule was triggered, then the workflow will be executed upon
-  update. A similar behaviour triggers a time schedule, if the time has passed for when it should have been executed.
+- *Create*: This happens either as part of application deployment or through the Lifecycle HTTP
+  RESTful API. After creating a schedule, it is initially disabled and will not execute any jobs.
+- *Disable*: Disabling a schedule will delete all pending jobs for the schedule from the job
+  queue, and prevent new jobs from being created. This action will not suspend or abort any
+  current execution of the workflow.
+- *Enable*: This action will put the schedule back into an active state, after a *Disable*
+  action. Note that if the schedule was previously disabled, that aborted all pending jobs
+  for the schedule. Therefore new triggers have to create new jobs for this schedule before
+  its workflow is executed again.
+- *Delete*: This first disables the schedule and then permanently deletes it.
+- *Update*: This is equivalant to deleting the current schedule and creating a new one. It happens
+  either when an application which contains a schedule is redeployed in CDAP, or through the
+  Lifecycle HTTP RESTful API.
+
+.. rubric:: Application Deployment and Schedules
+
+Schedules can be defined as part of application deployment, by calling ``schedule()``
+in the application's ``configure()`` method; or they can be managed separately from
+application deployment through the HTTP RESTful API. This can create a dilemma: After
+modifiying schedules through REST, redeploying the application, which may happen for
+reasons unrelated to the schedules, would undo all those changes and reinstate the schedules
+defined by the ``configure()`` method. Because that is undesired, CDAP provides an option
+to configure whether schedules are controlled by the ``configure()`` method or not. This
+option is called ``app.deploy.update.schedules`` and is given as a field of the
+:ref:`application deployment request <http-restful-api-lifecycle-create-app>`.
+
+- If this option is true, then application deployment will replace all schedules for
+  the application with the schedules given by the ``configure()`` method;
+- if it is false, application deployment will not change any schedules, except that if
+  the application is updated, all schedules for programs that do not exist any longer
```

Comment:
you mention that schedules removed from the app#configure are an exception.
What about schedules that are new in the new app#configure?